index.js

```diff
@@ -49,5 +50,30 @@ module.exports = {
   format: function(text, opts) {
     return formatWithShebang(text, normalizeOptions(opts));
   },
-  version: version
+  version: version,
+
+  __debug: {
```

Comment:
@jlongster I don't really want it to be part of the API but I need it for the docs and the CLI. I figured this was easier to put here than to try and hide it. Let me know if you think it's okay.